### DISCUSSION TITLE
(Question) How to change the background-color of notes?

### DISCUSSION THREAD

**Discussion by xzmail** [TriliumNext/Trilium#6323] [Q&A] [answered]
![image](https://github.com/zadam/trilium/assets/131539321/8cdd10dd-18fe-4d1b-8a69-2d7451b7f7e9)
i could change the color of the text in tree view, but cant find the way to change the background-color of the each note indvidually.

please help.

**Comment by azat385** [comment]
the same question!

**Comment by eliandoran** [comment, marked as answer]
Add `#cssClass=foo` where `foo` can be any name, and then add a CSS note with `#appCss` with the following:

```
.fancytree-node.hi {
  background-color: red !important;
}
```
